catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcTable.java

```diff
@@ -4,30 +4,65 @@
  */
 package com.datastrato.gravitino.catalog.jdbc;
 
+import com.datastrato.gravitino.catalog.jdbc.operation.TableOperation;
 import com.datastrato.gravitino.connector.BaseTable;
 import com.datastrato.gravitino.connector.TableOperations;
+import com.datastrato.gravitino.rel.SupportsPartitions;
 import com.google.common.collect.Maps;
 import java.util.Map;
-import lombok.Getter;
 import lombok.ToString;
 
 /** Represents a Jdbc Table entity in the jdbc table. */
 @ToString
-@Getter
 public class JdbcTable extends BaseTable {
+  private String databaseName;
+  private TableOperation tableOperation;
 
   private JdbcTable() {}
 
   @Override
   protected TableOperations newOps() {
-    // TODO: Implement this method when we have the JDBC table operations.
-    throw new UnsupportedOperationException("JdbcTable does not support TableOperations.");
+    if (tableOperation == null) {
```

Comment:
Currently, we only use `tableOperation` to support table partition operations, so I only set `tableOperation` in `JdbcCatalogOperations#loadTable`. Under the current design, in fact the `tableOperation` here will never be null, this is just to prevent wrong calls. 
BTW, We may need `tableOperation` elsewhere in the future, it seems reasonable to set `tableOperation` anywhere we create an instance of `JdbcTable`, In this way we will not need to consider the null situation, what's your opinion?

Reply:
Yeah, I think so.  `tableOperation` should not be null and we can use other ways to indicate that partition is not supported.